src/aiida/cmdline/commands/cmd_presto.py

```diff
@@ -0,0 +1,133 @@
+###########################################################################
+# Copyright (c), The AiiDA team. All rights reserved.                     #
+# This file is part of the AiiDA code.                                    #
+#                                                                         #
+# The code is hosted on GitHub at https://github.com/aiidateam/aiida-core #
+# For further information on the license, see the LICENSE.txt file        #
+# For further information please visit http://www.aiida.net               #
+###########################################################################
+"""``verdi presto`` command."""
+
+from __future__ import annotations
+
+import pathlib
+import re
+import typing as t
+
+import click
+
+from aiida.cmdline.commands.cmd_verdi import verdi
+from aiida.cmdline.utils import echo
+from aiida.manage.configuration import get_config_option
+
+DEFAULT_PROFILE_NAME_PREFIX: str = 'presto'
+
+
+def default_profile_name():
+    from aiida.manage import get_config
+
+    profile_names = get_config().profile_names
+    indices = []
+
+    for profile_name in profile_names:
+        if match := re.search(r'presto[-]?(\d+)?', profile_name):
+            indices.append(match.group(1) or '0')
+
+    if not indices:
+        return DEFAULT_PROFILE_NAME_PREFIX
+
+    last_index = int(sorted(indices)[-1])
+
+    return f'{DEFAULT_PROFILE_NAME_PREFIX}-{last_index + 1}'
+
+
+@verdi.command('presto')
+@click.option(
+    '--profile-name',
+    default=lambda: default_profile_name(),
+    show_default=True,
+    help='The name of the profile. By default generates a name that does not already exist and starts with '
+    f'`{DEFAULT_PROFILE_NAME_PREFIX}`.',
+)
+@click.option(
+    '--email',
+    default=get_config_option('autofill.user.email') or 'aiida@localhost',
+    show_default=True,
+    help='The email of the default user.',
+)
+@click.pass_context
+def verdi_presto(ctx, profile_name, email):
+    """Set up a new profile as automatically as possible.
+
+    This command aims making setting up a new profile as easy as possible. It intentionally provides a limited amount of
```

Comment:
Whatever we end up with in the reference documentation should be copied to this docstring.

Reply:
It is actually the reference doc that is automatically generated from this docstring here in a pre-commit step. So I applied your suggestions to the code here and then the docs should be automatically synced.